pkg/products/grafana/reconciler.go

```diff
@@ -350,3 +368,37 @@ func RandStringRunes(s int) string {
 	b := generateRandomBytes(s)
 	return base64.URLEncoding.EncodeToString(b)
 }
+
+func (r *Reconciler) isRatelimitJsonChanged(ctx context.Context, client k8sclient.Client, newRateLimitJsonStr string) bool {
```

Comment:
Remove please and let it be done by the CreateOrUpdate

Reply:
done. Thank you